images/Dockerfile

```diff
@@ -15,16 +15,16 @@ RUN apk add --no-cache \
     git \
     make \
     musl-dev
-ADD . $GOPATH/src/github.com/IBM-Blockchain/bcdb-server
-WORKDIR $GOPATH/src/github.com/IBM-Blockchain/bcdb-server
+ADD . $GOPATH/src/github.com/hyperledger-labs/orion-server
+WORKDIR $GOPATH/src/github.com/hyperledger-labs/orion-server
 
-FROM golang as bcdb
+FROM golang as orion-server
 RUN make binary
 
-FROM bcdb-base
-VOLUME /etc/bcdb-server
-VOLUME /var/bcdb-server
-COPY --from=bcdb /go/src/github.com/IBM-Blockchain/bcdb-server/bin/bdb /usr/local/bin
-COPY --from=bcdb /go/src/github.com/IBM-Blockchain/bcdb-server/sampleconfig/ /etc/bcdb-server
+FROM orion-base
+VOLUME /etc/orion-server
+VOLUME /var/orion-server
+COPY --from=orion-server /go/src/github.com/hyperledger-labs/orion-server/bin/bdb /usr/local/bin
+COPY --from=orion-server /go/src/github.com/hyperledger-labs/orion-server/sampleconfig/ /etc/bcdb-server
```

Comment:
bcdb-server -> orion-server. It doesn't matter too much, because volume configuration in `docker run` will override it in any case, but it rare case user runs docker without volume mapping, it will cause some problems.

Reply:
good catch